test/unittests/parser_expr_test.cpp

```diff
@@ -320,33 +313,34 @@ TEST(parser_expr, instr_br_table)
             (block
               (block
                 (br_table 3 2 1 0 4 (get_local 0))
-                (return (i32.const 99))
+                (return (i32.const 0x41))
```

Comment:
Any particular reason these were renumbered? Just to make it easier to read below?